relay/network_graph/graph.py

```diff
@@ -548,36 +548,66 @@ def dump(self):
                              'Creditline BA': account.reverse_creditline})
         return output.getvalue()
 
-    def find_path(self, source, target, value=None, max_hops=None, max_fees=None, timestamp=None):
-        """
-        find path between source and target
-        the shortest path is found based on
-            - the number of hops
-            - the imbalance it adds or reduces in the accounts
-        """
+    def find_path_sender_pays_fees(self, source, target, value=None, max_hops=None, max_fees=None, timestamp=None):
+
+        cost, path = self._find_path(
+            source=target,  # we are searching path from target to source, to accumulate fees correctly.
+            target=source,
+            value=value,
+            max_hops=max_hops,
+            max_fees=max_fees,
+            timestamp=timestamp,
+            cost_accumulator_function=SenderPaysCostAccumulatorSnapshot,
+        )
+
+        return cost, list(reversed(path))
+
+    def find_path_receiver_pays_fees(self, source, target, value=None, max_hops=None, max_fees=None, timestamp=None):
+
+        return self._find_path(
+            source=source,
+            target=target,
+            value=value,
+            max_hops=max_hops,
+            max_fees=max_fees,
+            timestamp=timestamp,
+            cost_accumulator_function=ReceiverPaysCostAccumulatorSnapshot,
+        )
+
+    def _find_path(
```

Comment:
lets maybe use `_find_transfer_path` to be clear?

Reply:
Then it would be also `find_transfer_path_receiver_pays_fees`

Reply:
Ok, I will change it.